lib/plugins/aws/info/display.js

```diff
@@ -12,7 +12,15 @@ module.exports = {
     message += `${chalk.yellow('service:')} ${info.service}\n`;
     message += `${chalk.yellow('stage:')} ${info.stage}\n`;
     message += `${chalk.yellow('region:')} ${info.region}\n`;
-    message += `${chalk.yellow('stack:')} ${info.stack}`;
+    message += `${chalk.yellow('stack:')} ${info.stack}\n`;
+    message += `${chalk.yellow('resources:')} ${info.resourceCount}`;
+
+    if (info.resourceCount >= 150) {
```

Comment:
Would be good to verify if it has the object key like this `if (_.has(info, 'resourceCount') && info.resourceCount >= 150)`